server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileRef.java

```diff
@@ -132,14 +156,66 @@ public static QProfileRef from(@Nullable String key, @Nullable String lang, @Nul
     return fromName(lang, name);
   }
 
+  /**
+   * @deprecated provide organization
+   */
+  @Deprecated
   public static QProfileRef fromKey(String key) {
     return new QProfileRef(requireNonNull(key), null, null);
   }
 
+  /**
+   * @deprecated provide organization
+   */
+  @Deprecated
   public static QProfileRef fromName(String lang, String name) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)